test/framework/src/test/java/org/elasticsearch/test/test/SuiteScopeClusterIT.java

```diff
@@ -44,14 +41,8 @@
     public void testReproducible() throws IOException {
         if (ITER++ == 0) {
             CLUSTER_SEED = cluster().seed();
-            for (int i = 0; i < SEQUENCE.length; i++) {
```

Comment:
I'm not sure if this is ok to remove.

Reply:
I honestly haven't thought enough about it to be sure.

Reply:
I'd hunt down the original author before dropping it. I think the best thing to do at this point is mark it as `@AwaitsFix`.